Migration of account generation page to support pages (#3676)
w3f/polkadot-wiki: docs/learn/learn-account-generation.md

---

An address is the public part of a Polkadot account. The private part is the key used to access this
address. The public and private parts together make up a Polkadot account. You can think of the public address  of your account like your mailbox and the private key like the key to open that mailbox. Anybody can send mail to your mailbox, but only you can access them as only you have access to its key. In the context of Polkadot accounts,  anybody can send tokens to your public address but only you can transact with them using your private key. That is why you should keep your private key secret.

There are several ways to generate a Polkadot account:

> - [Polkadot{.js} Browser Extension](#polkadot-js-apps) **RECOMMENDED FOR MOST USERS**

Reviewer: Have you mistakenly pointed this ref link to point to Polkadot JS Apps?

```suggestion
- [Polkadot-JS Browser Extension](#polkadot-js-browser-extension) **RECOMMENDED FOR MOST USERS**
```

---

An address is the public part of a Polkadot account. The private part is the key used to access this

> address. The public and private parts together make up a Polkadot account. You can think about your public address like your mailbox and your private key like your key to open your mailbox. Everybody can send you information to your public address, but only you can access the box and read such information. In other words, everybody can know your public address but you should keep your private key secret.

Reviewer: I'm thinking if there are better words to to explain this than just saying public and private part.

Rephrased it a little bit. Feel free to edit it further

```suggestion
address. The public and private parts together make up a Polkadot account. You can think of the public address  of your account like your mailbox and the private key like the key to open that mailbox. Anybody can send mail to your mailbox, but only you can access them as only you have access to its key. In the context of Polkadot accounts,  anybody can send tokens to your public address but only you can transact with them using your private key. That is why you should keep your private key secret.
```

understand the risks of the account generation method you choose and how to properly mitigate them
in order to keep your funds safe.

:::info Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.

:::

## Polkadot JS Browser Extension

> The Polkadot JS Extension provides a reasonable balance of security and usability. It provides a
> separate local mechanism to generate your address and interact with Polkadot.

Reviewer: Staying consistent with Glossary https://support.polkadot.network/support/solutions/articles/65000110873-glossary and at the same time, preserving the link to this section referenced on third party websites
```suggestion
The Polkadot-JS Browser Extension (simply referred to as Polkadot Extension) provides a reasonable balance of security and usability. It provides a
separate local mechanism to generate your address and interact with Polkadot.
```

> This method involves installing the Polkadot JS extension and using it as a “virtual vault," separate

Reviewer: ```suggestion
This method involves installing the Polkadot Extension and using it as a “virtual vault," separate
```

from your browser, to store your private keys. It also allows the signing of transactions and
similar functionality.

It is still running on the same computer you use to connect to the internet with and thus is less
secure than using Parity Signer or other air-gapped approaches.

> :::info For the tutorial on how to create an account using Polkadot JS browser extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account). 

Reviewer: ```suggestion
:::info For the tutorial on how to create an account using Polkadot Extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account). 
```

### Set Address for Polkadot Mainnet

In Polkadot you can use the same address on multiple chains. To switch between chains you can follow the guidelines in [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-). The [Accounts page](learn-accounts.md#address-conversion-tools) has a tool you can use to
convert your address between the different chain formats. 

:::info For privacy reasons, we recommend creating a new address for each chain you're using.

> 

Reviewer: Reasoning behind the recommendation 

```suggestion

Your address' format is only visual - the data used to derive this representation of your address
are the same, so you can use the same address on multiple chains. However, for privacy reasons, we
recommend creating a new address for each chain you're using.
```
